src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -175,7 +182,7 @@ export const canShow = async ({
 
 	const response = await getBodyEnd(
 		contributionsPayload,
-		`${contributionsServiceUrl}/epic${queryString}`,
+		`http://localhost:8082/epic${queryString}`,
```

Comment:
This is always really fiddly as we need it when running locally, but we'll don't want this change